src/higlass/tilesets.py

```diff
@@ -134,7 +137,7 @@ def multivec(filepath: str, uid: str):
     )
 
 
-@hash_absolute_filepath_as_default_uid
+@hash_file_as_default_uid
 def cooler(filepath: str, uid: str):
```

Comment:
Still need to update the first arg of whichever of these tileset loaders support file-likes.

Reply:
So I actually intentionally didn't update those. The reason is that if you use rhodius you can use file pointers but if you use clodius you can't. Rhodius is unofficial right now so I haven't updated it.

Until we merge the changes in rhodius into clodius I think it's better to keep the top level functions as strings.